targets/frontend/package.json

```diff
@@ -16,31 +16,41 @@
     "@socialgouv/cdtn-slugify": "^4.30.0",
     "@socialgouv/cdtn-sources": "^4.30.0",
     "@socialgouv/matomo-next": "^1.1.2",
+    "@socialgouv/react-ui": "^4.17.0",
+    "@urql/core": "^1.12.3",
     "@zeit/next-source-maps": "0.0.4-canary.1",
     "argon2": "^0.27.0",
+    "cookie": "^0.4.1",
+    "d3": "^6.1.1",
+    "d3-hierarchy": "^2.0.0",
     "diff": "^4.0.2",
     "graphql": "^15.3.0",
     "http-proxy-middleware": "^1.0.5",
     "isomorphic-unfetch": "^3.0.0",
     "jsonwebtoken": "^8.5.1",
+    "memoizee": "^0.4.14",
     "next": "^9.5.3",
     "next-transpile-modules": "^4.1.0",
     "next-urql": "^1.1.0",
     "nodemailer": "^6.4.11",
     "polished": "^3.6.7",
     "react": "^16.13.1",
+    "react-autosuggest": "^10.0.2",
     "react-dom": "^16.13.1",
     "react-hook-form": "^6.8.6",
     "react-icons": "^3.11.0",
     "react-is": "^16.13.1",
+    "react-sortable-hoc": "^1.11.0",
     "semver": "^7.3.2",
     "sentry-testkit": "^3.2.1",
+    "styled-components": "^5.1.1",
```

Comment:
can we remove it  ? 
we already have emotion, I think 2 css-in-js lib is not necessary.

Reply:
Malheureusement c'est necessaire pour afficher les icones de l'UI cdtn :/

Reply:
c'est relou, surtout que les icones sont pas lié a styled components :(

Reply:
Oui c'est vrai qu'en y regardant bien il ne devrait pas y avoir de problème à l'import des icones uniquement ! Mais ça crash bien comme il faut que j'essaye sans pourtant :(